### ISSUE TITLE
BUG: Scalar field plotter causes MDANSE to crash on Windows

### ISSUE THREAD

**Issue by RastislavTuranyi** [ISISNeutronMuon/MDANSE#30]
In 2D/3D Plotter, when 'Plot in new window' is clicked with Scalar field plotter selected, MDANSE freezes and after several seconds closes.

**Comment by eurydice76** [comment]
The problem also happens with the Elevation plotter selected. I suspect that all the Plotter3D based plotters are impacted. The crash occurs when the page is added to the tab.

**Comment by RastislavTuranyi** [comment]
I have traced the crash to l[ine 797](https://github.com/ISISNeutronMuon/MDANSE/blob/ffe60e4f78e78da198cb58b1091f777d9a611bac/Src/GUI/Plugins/Plotter3D.py#L797) of the Plotter3D.py file:
```
def center_on_actor(self,actor, out=False):#TODO out is useless : remove param
        Xmin,Xmax,Ymin,Ymax,Zmin,Zmax = actor.GetBounds()
        Xavg = (Xmin+Xmax)/2. 
        Yavg = (Ymin+Ymax)/2.
        Zavg = (Zmin+Zmax)/2.
        self.camera.SetFocalPoint(Xavg, Yavg, Zavg)
        self.camera.SetPosition(Xavg, Yavg, 3*Zmax)
```
This function is indeed used by all the 3D plotters, but I have not experienced any issues with any of them other than the Scalar Field one.

Given that we know that the newer versions of VTK (7.x and higher) introduce issues in MDANSE, perhaps this is one of these?

**Comment by eurydice76** [comment]
Yeap. I can give a try with an older version of vtk and see. However, using the python debugger, I find the error at line 287 of the PlotterPlugin.py file. Very strange

**Comment by eurydice76** [comment]
I tried on a machine installed through conda (see [wiki](https://github.com/ISISNeutronMuon/MDANSE/wiki/Installing-MDANSE-from-source-on-ubuntu@focal-using-conda-pip)) and version 6.3 of vtk and 3.0.2 of wxPython and surprisingly I have no crash. I can plot with all 3D plots available in MDANSE (elevation, isosurface, scalar field).

**Comment by RastislavTuranyi** [comment]
I have tested this with the executable generated by CI/CD on Ubuntu 21.04 and it works there too; no crashes. It seems that this is an issue only on Windows; whether the current wheels or conda VTK and wxPython, the issue persists. @sanghamitra-mukhopadhyay, can you please use the CI/CD installer to check if this occurs on MacOS? If not, then this is indeed a purely Windows issue.

**Comment by RastislavTuranyi** [comment]
This issue has been resolved in #61. Please see [this reply](https://github.com/ISISNeutronMuon/MDANSE/pull/61#issuecomment-1112177175).
